src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/Models/DependencyModel.cs

```diff
@@ -9,45 +9,64 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.Models
 {
     internal abstract class DependencyModel : IDependencyModel
     {
+        [Flags]
+        private enum DependencyFlags
```

Comment:
Not sure if it would help due to field packing, but consider making this have an underlying type of `byte`?